src/Tokenizer/Analyzer/ClassAnalyzer.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tokenizer\Analyzer;
+
+use PhpCsFixer\Tokenizer\Analyzer\Analysis\TypeAnalysis;
+use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\Tokenizer\TokensAnalyzer;
+
+/**
+ * @internal
+ */
+final class ClassAnalyzer
+{
+    /**
+     * @param Tokens $tokens
+     * @param int    $classIndex
+     *
+     * @return array
+     *               Contains information about the class being analyzed
```

Comment:
What is the reason of having the description in separate line?

Reply:
Probably a mistake, I fixed it.